addons/stock/static/src/js/report_stock_forecasted.js

```diff
@@ -101,14 +101,21 @@ const ReplenishReport = clientAction.extend({
             method: 'fields_view_get',
             kwargs: {
                 view_type: 'graph',
-            }
-        }).then(viewInfo => {
+            },
+        });
+        const productIds = this._rpc({
+            model: 'product.product',
+            method: 'get_components',
+            args: [[this.productId]],
+        });
+        let data = Promise.all([promController, productIds]);
+        data.then(res => {
```

Comment:
instead of using a array + index. Maybe you can use the "Destructuring Assignment" syntax to be more readable:
```javascript
data.then(([ (viewInfo, products_ids) ]) => {
   // ...
}
```